sdk/cosmosdb/cosmos/src/common/constants.ts

```diff
@@ -265,6 +282,8 @@ export const Constants = {
   // Changefeed AllVersionsAndDeletesMode formatting version
   AllVersionsAndDeletesChangeFeedWireFormatVersion: "2021-09-15",
   ChangeFeedIfNoneMatchStartFromNowHeader: "*",
+  // Default TTL for caches related to Encryption
+  DefaultEncryptionCacheTimeToLive: 2,
```

Comment:
Add comment that this will be used in hours

Reply:
added